### PR TITLE
[VL] Fix query benchmark

### PR DESCRIPTION
**Pull request by rui-mo**
## What changes were proposed in this pull request?

Removed Q1/Q6 query test, and import a selecting test.

## How was this patch tested?

under test

### CODE REVIEW

File: cpp/velox/benchmarks/QueryBenchmark.cc [hunk 1]
```diff
@@ -26,7 +26,8 @@ auto BM = [](::benchmark::State& state,
              const std::vector<std::string>& datasetPaths,
              const std::string& jsonFile,
              const std::string& fileFormat) {
-  const auto& filePath = getExampleFilePath("plan/" + jsonFile);
+  // const auto& filePath = getExampleFilePath("plan/" + jsonFile);
+  const auto& filePath = "/home/morui/Downloads/gluten/cpp/velox/benchmarks/data/plan/select.json";
```
Comment (zuochunwei) [inline cpp/velox/benchmarks/QueryBenchmark.cc:30]: the path contains your name, use `env-variable` or `relative-path` instead?

File: cpp/velox/benchmarks/QueryBenchmark.cc [hunk 2]
```diff
@@ -66,31 +67,14 @@ int main(int argc, char** argv) {
   // The multi-thread performance is not correct.
   // BENCHMARK(BM)->ThreadRange(36, 36);
 
-  // Register for TPC-H Q1 ORC tests.
-  std::string lineitemOrcPath = getExampleFilePath("orc/bm_lineitem/");
+  // std::string lineitemParquetPath = getExampleFilePath("bm_lineitem/parquet");
+  std::string lineitemParquetPath = "/home/morui/Downloads/gluten/cpp/velox/benchmarks/data/bm_lineitem/parquet/";
```
Comment (zuochunwei) [inline cpp/velox/benchmarks/QueryBenchmark.cc:71]: contain morui

File: cpp/velox/benchmarks/QueryBenchmark.cc (hunk 1 above)
Comment (rui-mo) [reply cpp/velox/benchmarks/QueryBenchmark.cc:30]: Sure. It's only a draft now, will continue on the improvement.

Comment (rui-mo) [reply cpp/velox/benchmarks/QueryBenchmark.cc]: Fixed.

**Comment by rui-mo** [comment]
Hi @zuochunwei, just added both Parquet and ORC files. Previous file was too large to be put into a repo, so it was replaced. Please take a look, thanks!

**Review by zuochunwei — APPROVED** [review]

**Review by zuochunwei — APPROVED** [review]

**Pull request merged**